pkg/vex/vex.go

```diff
@@ -65,15 +64,17 @@ func statementsFromAdvisories(advisories build.Advisories, purls []string) []vex
 	var stmts []vex.Statement
 
 	for v, entries := range advisories {
-		for _, entry := range entries {
-			stmts = append(stmts, statementFromAdvisoryContent(entry, v, purls))
+		for i := range entries {
+			stmts = append(stmts, statementFromAdvisoryContent(&entries[i], v, purls))
```

Comment:
Why change to using index integers instead of using the slice elements directly?